music-journal/components/AddMemory.js

```diff
@@ -17,21 +28,21 @@ const AddMemory = (props) => {
             <Row className="justify-content-center text-center">
                 <Form>
                     <Row className="justify-content-center">
-                        <Form.Group>
+                        <Form.Group controlId="formSong">
                             <Form.Label>Write a song name</Form.Label>
-                            <Form.Control id="song" as="textarea" rows="1" />
+                            <Form.Control as="textarea" onChange={({target}) => setSong(target.value)} rows="1" />
```

Comment:
so `target` in this case is whatever the input of the form is?

Reply:
Yeah so the object is event.target.value which gets the value of that input but you can use ({target}) to unwrap it and make the jsx a bit tidier. Text fields are easy but radio buttons etc are a bit more complicated